.github/workflows/package.yml

```diff
@@ -0,0 +1,95 @@
+name: package
+on: [push, pull_request]
+jobs:
+  linux-wheels:
+    strategy:
+      matrix:
+        arch: [x86, x86_64, arm64, armhf, mips]
```

Comment:
For consistency with other CI it would be good to order the archs like in the frida repo.